### ISSUE TITLE
Depth estimation example on v1.3.0 vs v1.2

### ISSUE THREAD

**Issue by ocularbioengarhku** [Unity-Technologies/sentis-samples#11]
## May I know what is the suggested version to run the depth estimation example?

System requirement (just to recap):
- Unity Sentis 1.0-1.2
   - Unity 2021.3+
- Unity Sentis 1.3.0-pre.3
   - Unity 2023.2+

### SITURATION
When following the [code ](https://github.com/Unity-Technologies/sentis-samples/tree/main/DepthEstimationSample) to open the project in Unity 2023.2.0b17, the function work perfectly. However, as my project includes the Google Cardboard package, and which **only available in <= Unity 2023.1.20, so only Sentis 1.2 can be used.**

## ERROR: depth estimation example in Unity Sentis 1.2
When open the depth estimation example in Unity Sentis 1.2 there is a error message in the `InferenceWebcam` script, `TextureConverter.ToTensor()`:
 
`Argument 2: cannot convert from 'Unity.Sentis.TensorFloat' to 'int'`

### SOLUTION
After some time of research, I solved the issue and share here, in case someone is trying to do so.

Sentis 1.2 is missing this void ToTensor function, which is newly add to the Sentis 1.3.0.
`ToTensor (Texture texture, TensorFloat tensor, TextureTransform transform): Void`

Therefore, just add this function to the package script. This [link ](https://support.unity.com/hc/en-us/articles/9113460764052-How-can-I-modify-built-in-packages-) will show you how to modify the package. 

## Sentis 1.3.0 or 1.2?
So in conclusion, which version is suggested to use for depth estimation? Thank you!

**Comment by LiutaurasVy** [comment]
We're currently focusing on supporting Unity 2023.3 and the latest Sentis versions. Unfortunately, that means we won't be pushing an official fix to support other Unity/Sentis versions.

**Comment by ocularbioengarhku** [comment]
Ok, got it. Thank you for your information!
